apps/api/v2/src/modules/webhooks/outputs/webhook.output.ts

```diff
@@ -36,6 +36,10 @@ export class WebhookOutputDto {
   @IsBoolean()
   @Expose()
   readonly active!: boolean;
+
+  @IsString()
+  @Expose()
+  readonly secret?: string;
```

Comment:
I noticed that on line 30 we have `@IsEnum(WebhookTriggerEvents)` but triggers is of type `WebhookTriggerEvents[]`